updates/version.yaml

```diff
@@ -3,3 +3,6 @@
 1.0.2:
     - 'Created table suresoftware_maillog_log'
     - builder_table_create_suresoftware_maillog_log.php
+1.0.3:
```

Comment:
Since this is a fairly big improvement to the plugin, I'd say this commends a 1.1.0 version bump :)

Reply:
Agree, version updated !